src/components/pages/blogs/blog/components/TableTemplate.tsx

```
@@ -19,58 +19,68 @@ interface ITableTemplate {
-          key={index}
-          className="font-normal text-base leading-8 my-5 text-black"
-          dangerouslySetInnerHTML={{ __html: item }}
-        />
-      ))}
+      {/* Top Description */}
+      {content.topDescription && (
+        <div className="mb-8">
+          {content.topDescription.split("\n").map((paragraph, index) => (
+            <p key={index}
+              className="text-gray-700 leading-relaxed mb-4 text-base lg:text-lg"
+              dangerouslySetInnerHTML={{ __html: paragraph }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)